phabricator.user.js

```diff
@@ -0,0 +1,19 @@
+// ==UserScript==
+// @name     Phabricator
+// @version  1.0
+// @grant    none
+// @require  https://code.jquery.com/jquery-3.4.1.slim.min.js
+// @include  https://phabricator.wikimedia.org/*
+// ==/UserScript==
+
+$(function() {
+    $("<a></a>")
+        .attr("href", "/notification/query/unread/")
+        .addClass("alert-notifications")
+        .append(
+            $("<span></span>")
+                .addClass("phabricator-main-menu-message-icon phui-icon-view phui-font-fa fa-bullhorn")
```

Comment:
The following changes the icon to have the same white colour as the bell.
```
       .addClass("alert-notifications alert-unread")
        .append(
            $("<span></span>")
                .addClass("phabricator-main-menu-alert-icon phui-icon-view phui-font-fa fa-bullhorn")
```

Reply:
This cannot be automatically applied (unlike the above ones) :(

Reply:
Fixed.